### PR TITLE
Cleanup aws-janitor part 2

### PR DESCRIPTION
**Pull request by vincepri**
This is a followup from @liztio's #10707.

- General readability cleanup
- Switches to klog
- Consistency fixes throughout the package
- Updates AWS SDK

Signed-off-by: Vince Prignano <vincepri@vmware.com>

### CODE REVIEW

File: maintenance/aws-janitor/main.go [hunk 1]
```diff
@@ -27,15 +27,19 @@ import (
...
 	"github.com/aws/aws-sdk-go/service/iam"
-	"github.com/golang/glog"
+	"k8s.io/klog"
 	"k8s.io/test-infra/maintenance/aws-janitor/resources"
 	s3path "k8s.io/test-infra/maintenance/aws-janitor/s3"
 )
 
-const defaultRegion = "us-east-1"
+const (
+	defaultRegion = "us-east-1"
```
Comment (liztio) [inline maintenance/aws-janitor/main.go]: any particular reason for this?

Comment (vincepri) [reply maintenance/aws-janitor/main.go]: I think this is one of the tool vscode has on save that reorders this and imports stmts.

**Comment by krzyzacy** [comment]
/approve

please rebase :-)

**Comment by vincepri** [comment]
All done!

**Comment by vincepri** [comment]
@liztio @rdodev need another lgtm, just rebased

File: maintenance/aws-janitor/main.go [hunk 2]
```diff
@@ -24,33 +24,42 @@ import (
...
-	var regions []string
 	svc := ec2.New(sess, &aws.Config{Region: aws.String(defaultRegion)})
+
 	resp, err := svc.DescribeRegions(nil)
 	if err != nil {
 		return nil, err
 	}
+
+	var regions []string
 	for _, region := range resp.Regions {
```
[the remark this answers is not in the document]
Comment (rdodev) [reply maintenance/aws-janitor/main.go]: Nevermind. Just realized this will only return the subset of regions available to the credentials.

**Comment by vincepri** [comment]
@rdodev all fixed!

**Comment by krzyzacy** [comment]
/lgtm
/approve
per https://github.com/kubernetes/test-infra/pull/10720#pullrequestreview-198826655

**Pull request merged**